addons/imagedecoder.heif/imagedecoder.heif.json

```diff
@@ -28,14 +20,22 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/strukturag/libheif/releases/download/v1.12.0/libheif-1.12.0.tar.gz",
-                    "sha256": "e1ac2abb354fdc8ccdca71363ebad7503ad731c84022cf460837f0839e171718"
+                    "url": "https://github.com/strukturag/libde265/releases/download/v1.0.8/libde265-1.0.8.tar.gz",
```

Comment:
We should probably try to at least match these? https://github.com/xbmc/imagedecoder.heif/blob/Omega/depends/common/libde265/libde265.txt

Reply:
`-DBUILD_SHARED_LIBS=OFF`  doesn't help for flatpak and may be negative for additional size. `-DWITH_LIBDE265=OFF` contradicts existing libde265 dependency. Better someone who added this explain what they wanted to achieve. I would rather not touch this myself.

Reply:
done